packages/financial-templates-lib/test/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -0,0 +1,163 @@
+const { toWei } = web3.utils;
+
+const { GasEstimator, SpyTransport, spyLogIncludes } = require("../../index");
+
+const { getTruffleContract } = require("@uma/core");
+
+const winston = require("winston");
+const sinon = require("sinon");
+
+// Script to test
+const { DSProxyManager } = require("../../src/proxy-wallet-transaction-handler/DSProxyManager.js");
+
+const TokenSender = getTruffleContract("TokenSender", web3);
```

Comment:
I think you should explicitly set "latest" for readability

Reply:
sure & done.